AGENTS.md

```diff
@@ -27,6 +27,9 @@ This is a blog developed using the Astro framework (https://astro.build/).
 - **Code quality**: ESLint + Prettier for linting and formatting
 - **Reproducibility**: Deterministic builds with pinned versions for security
 
+## Design & Content Guidelines
+- **Casing**: Use sentence case for all subtitles and descriptions (e.g., "A developer blog"). The only exception is the site title "All Bugs Are Fixed".
```

Comment:
The only exception is for proper nouns and acronyms. The sentence case is for all titles and headings